README.md

```diff
@@ -75,7 +75,7 @@ repo](https://github.com/rapidsai/notebooks-contrib).
 | | Truncated Singular Value Decomposition (tSVD) | Multi-GPU version available (CUDA 10 only) |
 | | Uniform Manifold Approximation and Projection (UMAP) | |
 | | Random Projection | |
-| | t-Distributed Stochastic Neighbor Embedding (TSNE) | (Experimental) |
+| | t-Distributed Stochastic Neighbor Embedding (TSNE) | Barnes-Hut O(NlogN) and Exact O(N^2) |
```

Comment:
Nitpick- but I'm not quite sure we need to put asymptotic runtimes in this list. Listing Barnes-Hut and Exact versions should be enough. 

It would be more worthwhile to reference the Barnes-Hut paper in the PyDocs / Doxygen docs.

Reply:
Oh lol yep good points :) I'll update them

Reply:
Ok changed :) In regards to the papers, I think there's 3 references at the end of the Py docs.